frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/UploadImageModel.java

```diff
@@ -169,11 +174,26 @@ public void initialize() {
 
                     getDataCenter().setIsChangeable(isChangeable);
                     if (!isChangeable) {
-                        // Set the selected item on the storage domains list.
-                        AsyncDataProvider.getInstance().getStorageDomainById(
-                                new AsyncQuery<>(storageDomain -> {
-                                    getStorageDomain().setItems(Collections.singletonList(storageDomain));
-                                }), limitToStorageDomainId);
```

Comment:
`AsyncDataProvider.getInstance().getStorageDomainById ` queries Storage Domain details by using the `GetStorageDomainById` query which in turn uses the `getstorage_domains_by_id` function which uses `storage_domains_without_storage_pools` view. And the `storage_domains_without_storage_pools` view does not return the `storagePoolId` (Data Center id) property for the Storage Domain